### PR TITLE
adding NoSchedule taint troubleshooting documentation

### PR DESCRIPTION
**Pull request by raymond-zhang00**
*Issue #, if available:*
[1527](https://github.com/aws/eks-anywhere-internal/issues/1527)

*Description of changes:*
Adds documentation for users on the WorkerNodeGroup NoSchedule/NoExecute validation.

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/troubleshooting/troubleshooting.md [hunk 1]
```diff
@@ -76,6 +76,14 @@ Error: old cluster config file exists under my-cluster, please use a different c
 The `my-cluster` directory already exists in the current directory.
 Either use a different cluster name or move the directory.
 
+### At least one WorkerNodeGroupConfiguration must not have NoExecute and/or NoSchedule taints
+
+```
+Error: the cluster config file provided is invalid: at least one WorkerNodeGroupConfiguration must not have NoExecute and/or NoSchedule taints
+```
+At least one schedulable worker node for is required to run cluster administration components.
```
Comment (chrisdoherty4) [inline docs/content/en/docs/troubleshooting/troubleshooting.md:84]:
```suggestion
At least one schedulable worker node is required to run cluster administration components.
```

File: docs/content/en/docs/troubleshooting/troubleshooting.md [hunk 2]
```diff
@@ -76,6 +76,14 @@ Error: old cluster config file exists under my-cluster, please use a different c
...
+Remove NoExecute and/or NoSchedule taints from at least one WorkerNodeGroupConfiguration.
```
Comment (chrisdoherty4) [inline docs/content/en/docs/troubleshooting/troubleshooting.md:85]: Can we give minimal examples of valid and invalid configurations?

Comment (raymond-zhang00) [reply docs/content/en/docs/troubleshooting/troubleshooting.md:85]: Added section with valid and invalid worker node config

File: docs/content/en/docs/troubleshooting/troubleshooting.md [hunk 3]
```diff
@@ -76,6 +76,14 @@ Error: old cluster config file exists under my-cluster, please use a different c
 The `my-cluster` directory already exists in the current directory.
 Either use a different cluster name or move the directory.
 
+### At least one WorkerNodeGroupConfiguration must not have NoExecute and/or NoSchedule taints
```
[the remark this answers is not in the document]
Comment (raymond-zhang00) [reply docs/content/en/docs/troubleshooting/troubleshooting.md:79]: That makes sense, kept the title the same to match the error string itself.  But added some clarification that both taints need to be removed. Also added example

**Review by chrisdoherty4 — APPROVED** [review]

**Pull request merged**